railgun/manager/ingressapi.go

```diff
@@ -25,17 +25,17 @@ const (
 func negotiateIngressAPI(config *config.Config, client client.Client) (IngressAPI, error) {
 	var allowedAPIs []IngressAPI
 	candidateAPIs := map[IngressAPI]schema.GroupVersionResource{
-		NetworkingV1: schema.GroupVersionResource{
+		NetworkingV1: {
```

Comment:
This module looks like a redundancy of 
railgun/apis/configuration/v1
railgun/apis/configuration/v1beta1

can we do some cleanup between them ?

Reply:
This PR did nothing to change this module _except_ for removing this typedef because it was throwing lint errors (for redundancy). I would like to add no further scope to this PR please we can refactor further in later iterations.